CLUSTER.md

```diff
@@ -0,0 +1,31 @@
+# GTSFM on CLUSTER
+
+### How to Run GTSfM on a Cluster?
+
+GTSfM uses [SSHCluster](https://docs.dask.org/en/stable/deploying-ssh.html#dask.distributed.SSHCluster) module of [Dask.distributed](https://distributed.dask.org/en/stable/) library to allow utilizing clusters for GTSfM runs. This is a step-by-step guide on how to set up your machines for a successful run on a cluster.
```

Comment:
nit: "uses the SSHCluster module of..."

Reply:
nit: I would say either 
`Gtsfm allows using clusters for SfM execution via the SSHCluster module of Dask`
or
`Gtsfm uses the SSHCluster module of Dask to provide cluster-utilization functionality for SfM execution`

Reply:
I would also say "This readme is a step-by-step guide..."

Reply:
Done.

Reply:
Resolved, thanks!